extensions/kusto/config.json

```diff
@@ -1,6 +1,6 @@
 {
 	"downloadUrl": "https://github.com/Microsoft/sqltoolsservice/releases/download/v{#version#}/microsoft.sqltools.servicelayer-{#fileName#}",
-	"version": "3.0.0-release.92",
+	"version": "3.0.0-release.93",
```

Comment:
why are we bumping the version in the Kusto extension for SQL DB migrations?

Reply:
oh I missed that, sorry! I must have accidentally changed the wrong config file